File: maxConsecutiveOne.c
int findMaxConsecutiveOnes(int* nums, int numsSize){
    int ones = 0, result = 0;
    for(int i=0; i<numsSize; i++){
        if (nums[i]==0){
            ones = 0;
        }
        else{
            ones+=1;
            if (ones>result){
                result = ones;
            }
        }
    }
    return result;
}

File: Max Consecutive Ones/maxConsecutiveOne.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        self.nums = nums
        count = 0
        ones = 0
        for i in range (len(self.nums)): 
            if self.nums[i] == 0:  
                count = 0
            else:
                count+=1
                ones = max(ones,count)
        return ones
